### PR TITLE
Reworked exercises to fit videos

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: src/chapters/thymeleaf-views/exercises.rst [hunk 1]
```diff
@@ -5,12 +5,36 @@ In the chapter, we started working on an application for tracking various coding
...
 
-#. In addition to names of events, we want to add descriptions as well.
-   Use fragments to store the name and description of different coding events around town.
-   Use ``th:replace`` in your main template to bring in the descriptions and names for each event in the list.
+Getting Started
+---------------
+
+Checkout a new branch off of ``master`` called ``my-exercises-solution``.
+
+Now for some Git magic! We are going to go back in time to when our templates were using data from a static event list.
```
Comment (jimflores5) [inline src/chapters/thymeleaf-views/exercises.rst]: How about "...were still using data..."?

File: src/chapters/thymeleaf-views/exercises.rst [hunk 2]
```diff
@@ -5,12 +5,36 @@ In the chapter, we started working on an application for tracking various coding
...
+
+#. Enter our ``git log`` command. Go through the logs until you find the commit where you finished creating a static event list.
```
Comment (jimflores5) [inline src/chapters/thymeleaf-views/exercises.rst]: "Enter the ``git log`` command."

(or "Use the ``git log`` command."

One potential problem.  What if a student made no commits between when they started the project and when they reached the exercises?

I added instructions to commit at the static list point in the book, but if someone does not, should there be a rescue option here?

File: src/chapters/thymeleaf-views/exercises.rst [hunk 3]
```diff
@@ -5,12 +5,36 @@ In the chapter, we started working on an application for tracking various coding
...
+
+   Before you reset to an older commit, make sure you are on a separate branch! This will reset your repo to a previous state!
+
+Now, let's add descriptions to our events!
+
+Expanding our Events Schedule
+-----------------------------
+
+#. Comment out your previous code in the ``displayAllEvents`` method.
+#. In the videos, wwe learned how to use templates to display the elements in a static list called ``events``. Let's make our ``events`` list a HashMap!
```
Comment (jimflores5) [inline src/chapters/thymeleaf-views/exercises.rst]: Typo: "... we learned..."

**Review by jimflores5 — APPROVED** [review]

File: src/chapters/thymeleaf-views/exercises.rst (hunk 1 above)
Comment (gildedgardenia) [reply src/chapters/thymeleaf-views/exercises.rst]: Done! Thanks!

File: src/chapters/thymeleaf-views/exercises.rst (hunk 2 above)
Comment (gildedgardenia) [reply src/chapters/thymeleaf-views/exercises.rst]: Added a rescue option that links back to Chris' repo!

File: src/chapters/thymeleaf-views/exercises.rst (hunk 3 above)
Comment (gildedgardenia) [reply src/chapters/thymeleaf-views/exercises.rst]: Done! Thanks!

**Pull request merged**